DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: simtrial
 Type: Package
 Title: Clinical Trial Simulation
-Version: 0.4.1.6
+Version: 0.4.1.7
```

Comment:
The current version is 0.4.1.8

https://github.com/Merck/simtrial/blob/69960550e90e11ff88bb210586eb48523403f32e/DESCRIPTION#L4

Reply:
Updated